src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/LocalHelpers.cs

```diff
@@ -52,6 +52,18 @@ public static string GetGitDir(ILogger logger)
             return dir;
         }
```

Comment:
I don't think this will/should be user anymore after your change, can you please remove it?

Reply:
Removed.

---
In reply to: [263629535](https://github.com/dotnet/arcade-services/pull/206#discussion_r263629535) [](ancestors = 263629535)